public/src/components/channelManagement/variantEditorSecondaryCtaEditor.tsx

```diff
@@ -56,8 +57,15 @@ const VariantEditorSecondaryCtaEditor: React.FC<VariantEditorSecondaryCtaEditorP
     <div className={classes.container}>
       <div className={classes.selectContainer}>
         <FormControl className={classes.formControl}>
-          <InputLabel>{label}</InputLabel>
-          <Select value={cta?.type || 'None'} onChange={handleChange} disabled={isDisabled}>
+          <InputLabel id="secondaryCtaTypeLabel">{label}</InputLabel>
+          <Select
+            id={'secondaryCtaType'}
+            labelId={'secondaryCtaTypeLabel'}
+            label={label}
+            value={cta?.type || 'None'}
+            onChange={handleChange}
+            disabled={isDisabled}
+          >
```

Comment:
The UI is a bit different here and this change ensures the label looks ok -
![Screenshot 2024-02-21 at 15 13 32](https://github.com/guardian/support-admin-console/assets/1513454/ebf7c256-479a-4d71-8882-e3d8e2455705)